crates/bevy_ecs/src/system/query.rs

```diff
@@ -834,6 +834,7 @@ where
     /// Panics if the number of query results is not exactly one. Use
     /// [`get_single`](Self::get_single) to return a `Result` instead of panicking.
     #[track_caller]
+    #[inline]
     pub fn single(&'s self) -> <Q::ReadOnlyFetch as Fetch<'w, 's>>::Item {
```

Comment:
Just as a helpful reminder: when resolving the "new" merge conflicts in this pr, make sure you choose
` pub fn single(&self) -> <Q::ReadOnlyFetch as Fetch<'_, 's>>::Item {`